descriptors/ModuleDescriptor-template.json

```diff
@@ -416,9 +416,71 @@
           "permissionsRequired": ["patron-action-session-storage.expired-session-patron-ids.collection.get"]
         }
       ]
+    },
+    {
+      "id": "check-in-storage",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/check-in-storage/check-ins",
+          "permissionsRequired": [
+            "check-in-storage.check-ins.collection.get"
+          ]
+        },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/check-in-storage/check-ins",
+          "permissionsRequired": [
+            "check-in-storage.check-ins.item.post"
+          ]
+        },
+        {
+          "methods": [
+            "DELETE"
+          ],
+          "pathPattern": "/check-in-storage/check-ins/{id}",
+          "permissionsRequired": [
+            "check-in-storage.check-ins.item.delete"
+          ]
+        },
```

Comment:
Why do we allow a check in event to be deleted?

Reply:
I was thinking that some time we will have a job that will remove "expired" records. But now I realized that a delete by a query will better suite here.

Should I remove this ability? What do you think?

Reply:
Will remove the endpoint, It should be easy enough to add once we need.